scripts/observers/config.py

```diff
@@ -125,6 +141,47 @@ def prepare_seed_node(working_dir):
     os.chmod(working_dir/"seednode/seednode", st.st_mode | stat.S_IEXEC)
 
 
+def prepare_proxy(working_dir):
+    print("preparing proxy")
+    mx_chain_proxy_go_folder = working_dir / "mx-chain-proxy-go"
+    subprocess.check_call(["go", "build"], cwd=mx_chain_proxy_go_folder / "cmd/proxy")
+
+    mx_chain_proxy_go_binary_folder = mx_chain_proxy_go_folder / "cmd/proxy"
+    st = os.stat(mx_chain_proxy_go_binary_folder / "proxy")
+    os.chmod(mx_chain_proxy_go_binary_folder / "proxy", st.st_mode | stat.S_IEXEC)
+
+    # prefs.toml
+    path_config = mx_chain_proxy_go_binary_folder / "config/config.toml"
+    config_data = toml.load(str(path_config))
+
+    proxy_port = int(os.getenv('PROXY_PORT'))
+    config_data['GeneralSettings']['ServerPort'] = proxy_port
+    del config_data['Observers']
+    del config_data['FullHistoryNodes']
+
+    config_data['Observers'] = []
+
+    observers_start_port = int(os.getenv('OBSERVERS_START_PORT'))
+    meta_observer = {
+        'ShardId': 4294967295,
+        'Address': f'http://127.0.0.1:{observers_start_port}',
+    }
+    config_data['Observers'].append(meta_observer)
+
+    num_of_shards = int(os.getenv('NUM_OF_SHARDS'))
+    for shardID in range(num_of_shards):
+        shard_observer_port = observers_start_port+shardID+1
```

Comment:
so the first port will be 9501, not 9500. can you change?

Reply:
the port 9500 is for the metachain observer/validator